app/src/main/AndroidManifest.xml

```diff
@@ -44,10 +44,14 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-            <intent-filter>
-                <data
-                    android:scheme="https"
-                    android:host="www.nowinandroid.apps.samples.google.com" />
+            <intent-filter android:autoVerify="true">
```

Comment:
I would advise against adding this parameter unless the Digital Asset Link is correctly setup.
This endpoint currently does not exists: https://www.nowinandroid.apps.samples.google.com./.well-known/assetlinks.json
The side effect of this would be to disable all currently associated deeplinks of this app (althrough there are none at the moment).
This also means that the proper flow -where the disambiguation dialog is not shown- is not allowed.

Reply:
Noted and removed 👍